### ISSUE TITLE
[BUG] - v2.3.6 - DM-NVX-360C reports Activation and Post-Activation errors

### ISSUE THREAD

**Issue by cdenig** [PepperDash/epi-crestron-nvx#108]
**Describe the bug**
DMNVx360C type devices are generating USB Routing errors during Activation.  This seems to lead to Post-Activation failures when creating the VideoRouting devices.

**To Reproduce**
1. Load Essentials with these devices configured
2. See errors in Activation and Port Activation stages

**Configuration**
```
{
        "key": "nvx-rx-3",
        "uid": 14,
        "name": "Audio A",
        "type": "DmNvx360C",
        "group": "nvx",
        "properties": {
          "control": {
            "method": "ipid",
            "ipid": "53"
          },
          "mode": "rx"
        }
      },
      {
        "key": "nvx-rx-4",
        "uid": 15,
        "name": "Codec A Content",
        "type": "DmNvxD30C",
        "group": "nvx",
        "properties": {
          "control": {
            "method": "ipid",
            "ipid": "54"
          },
          "mode": "rx"
        }
      },
      {
        "key": "nvx-rx-5",
        "uid": 17,
        "name": "Audio B",
        "type": "DmNvx360C",
        "group": "nvx",
        "properties": {
          "control": {
            "method": "ipid",
            "ipid": "55"
          },
          "mode": "rx"
        }
      },
```

**Stacktrace**
```
Update request from ID-97 in Program 01 
[10:43:32.034]App 1:Exception in GetUsbStream : Object reference not set to an instance of an object.
[10:43:32.037]App 1:[nvx-rx-3] Caught an exception in activate:System.NullReferenceException: Object reference not set to an instance of an object.  at NvxEpi.Services.Feedback.UsbRouteFeedback+<>c__DisplayClass5.<ReturnRoute>b__4 (NvxEpi.Devices.NvxBaseDevice o) [0x0000b] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate, System.Boolean& found) [0x0003f] in <3c04391f03a04620a28c80b17e22fd29>:0   at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x00000] in <3c04391f03a04620a28c80b17e22fd29>:0   at NvxEpi.Services.Feedback.UsbRouteFeedback.ReturnRoute (Crestron.SimplSharpPro.DM.Streaming.DmNvxBaseClass device) [0x000a7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Services.Feedback.UsbRouteFeedback.GetFeedback (Crestron.SimplSharpPro.DM.Streaming.DmNvxBaseClass device) [0x0000d] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Features.Streams.Usb.UsbStream..ctor (NvxEpi.Abstractions.INvxDeviceWithHardware device, System.Boolean isRemote, System.Boolean followStream, System.String defaultPair, System.Boolean isLayer3) [0x000e7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Features.Streams.Usb.UsbStream.GetUsbStream (NvxEpi.Abstractions.INvxDeviceWithHardware device, NvxEpi.Features.Config.NvxUsbProperties incomingProps) [0x000f7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Devices.Nvx36X.CustomActivate () [0x00025] in <11ce35af9c5143d1bb2fc144df022e0d>:0 
[10:43:32.038]App 1:[nvx-rx-3] ERROR: Device nvx-rx-3 Activation failure: Object reference not set to an instance of an object.
[10:43:32.040]App 1:Exception in GetUsbStream : Object reference not set to an instance of an object.
[10:43:32.041]App 1:[nvx-rx-5] Caught an exception in activate:System.NullReferenceException: Object reference not set to an instance of an object.  at NvxEpi.Services.Feedback.UsbRouteFeedback+<>c__DisplayClass5.<ReturnRoute>b__4 (NvxEpi.Devices.NvxBaseDevice o) [0x0000b] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate, System.Boolean& found) [0x0003f] in <3c04391f03a04620a28c80b17e22fd29>:0   at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x00000] in <3c04391f03a04620a28c80b17e22fd29>:0   at NvxEpi.Services.Feedback.UsbRouteFeedback.ReturnRoute (Crestron.SimplSharpPro.DM.Streaming.DmNvxBaseClass device) [0x000a7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Services.Feedback.UsbRouteFeedback.GetFeedback (Crestron.SimplSharpPro.DM.Streaming.DmNvxBaseClass device) [0x0000d] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Features.Streams.Usb.UsbStream..ctor (NvxEpi.Abstractions.INvxDeviceWithHardware device, System.Boolean isRemote, System.Boolean followStream, System.String defaultPair, System.Boolean isLayer3) [0x000e7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Features.Streams.Usb.UsbStream.GetUsbStream (NvxEpi.Abstractions.INvxDeviceWithHardware device, NvxEpi.Features.Config.NvxUsbProperties incomingProps) [0x000f7] in <11ce35af9c5143d1bb2fc144df022e0d>:0   at NvxEpi.Devices.Nvx36X.CustomActivate () [0x00025] in <11ce35af9c5143d1bb2fc144df022e0d>:0 
[10:43:32.042]App 1:[nvx-rx-5] ERROR: Device nvx-rx-5 Activation failure: Object reference not set to an instance of an object.
[10:43:32.044]App 1:****Activation complete****
[10:43:32.044]App 1:****PostActivation starting...****
.....[10:43:37.082]App 1:Register device result: 'nvx-tx-1', type 'DM-NVX-E30C IP-ID-31', result Success
[10:43:37.142]App 1:Register device result: 'nvx-tx-2', type 'DM-NVX-E30C IP-ID-32', result Success
[10:43:37.150]App 1:Register device result: 'nvx-tx-3', type 'DM-NVX-E30C IP-ID-33', result Success
[10:43:37.156]App 1:Register device result: 'nvx-tx-4', type 'DM-NVX-E30C IP-ID-34', result Success
[10:43:37.164]App 1:Register device result: 'nvx-tx-5', type 'DM-NVX-E30C IP-ID-35', result Success
[10:43:37.175]App 1:Register device result: 'nvx-tx-6', type 'DM-NVX-E30C IP-ID-36', result Success
[10:43:37.183]App 1:[nvx-rx-3--VideoRoutingRx] ERROR: Device nvx-rx-3--VideoRoutingRx PostActivation failure: Object reference not set to an instance of an object.
[10:43:37.184]App 1:Register device result: 'nvx-tx-7', type 'DM-NVX-E30C IP-ID-37', result Success
[10:43:37.187]App 1:[nvx-rx-5--VideoRoutingRx] ERROR: Device nvx-rx-5--VideoRoutingRx PostActivation failure: Object reference not set to an instance of an object.
[10:43:37.202]App 1:Register device result: 'nvx-tx-8', type 'DM-NVX-E30C IP-ID-38', result Success
[10:43:37.211]App 1:Register device result: 'nvx-tx-9', type 'DM-NVX-E30C IP-ID-39', result Success
[10:43:37.248]App 1:Register device result: 'nvx-tx-10', type 'DM-NVX-E30C IP-ID-3A', result Success
[10:43:37.261]App 1:Register device result: 'nvx-rx-1', type 'DM-NVX-D30 IP-ID-51', result Success
[10:43:37.268]App 1:Register device result: 'nvx-rx-2', type 'DM-NVX-D30 IP-ID-52', result Success
[10:43:37.287]App 1:Register device result: 'nvx-rx-3', type 'DM-NVX-360C IP-ID-53', result Success
[10:43:37.295]App 1:Register device result: 'nvx-rx-4', type 'DM-NVX-D30C IP-ID-54', result Success
[10:43:37.302]App 1:Register device result: 'nvx-rx-5', type 'DM-NVX-360C IP-ID-55', result Success
[10:43:37.309]App 1:Register device result: 'nvx-rx-6', type 'DM-NVX-D30C IP-ID-56', result Success
```

**Expected behavior**
USB errors not to be generated and if they are generated, not to affect creation of VidoeRouting device.

**Screenshots**
n/a

**Control Processor**
 - Device: CP4N
 - Version: assume latest, did not capture

**Additional context**
NA

**Comment by ngenovese11** [comment]
@cdenig sorry for the late response here, I have a fix for this pending review, thanks for posting

**Comment by ngenovese11** [comment]
closed as fixed in #109
